cluster/mesos/docker/docker-compose.yml

```diff
@@ -30,14 +30,15 @@ mesosslave1:
   hostname: mesosslave1
   privileged: true
   image: mesosphere/mesos-slave-dind:0.23.0-1.0.ubuntu1404.docker181
-  entrypoint: [ "bash", "-c", "wrapdocker mesos-slave --hostname=$(getent hosts mesosslave1 | cut -d' ' -f1 | sort -u | tail -1)" ]
   command: ~
   environment:
+  - MESOS_HOSTNAME=mesosslave1
```

Comment:
No, it was supported before, but we did not explicitly pass it to the kubelet (only indirectly as hostname). In production environment one might have real DNS names for slaves, not IPs. That's why I added it here.